beeai/agents/triage_agent.py

```diff
@@ -156,11 +164,14 @@ def prompt(self) -> str:
                (e.g. in the commit hash field or comment)
 
              2.2. Systematic Source Investigation
-             * Identify the official upstream project and corresponding Fedora package source
+             * Identify the official upstream project from two sources:
+                * Links from the Jira issue (if any direct upstream links are provided)
+                * Package specfile (<package>.spec) in the GitLab repository: check the URL field or Source0 field for upstream project location
```

Comment:
In theory we could provide a tool for this; how reliable was this step?

Reply:
I was wondering if there is something like that, I will leave it like this for now, but will check if there isn't something already; good point

Reply:
I found the tool I was talking about, it's dead. Not sure what was the source of information for it, I'm not aware of any DB/repo with this data.